### DISCUSSION TITLE
[question] 请问`三方工具库`是哪里进行导入呢？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#878] [Q&A] [answered]
在阅读文章的时候，看到tiny-engine可以将三方库导入进行使用：

请问`三方工具库`是哪里进行导入呢？

![image](https://github.com/user-attachments/assets/da559498-59eb-49bb-a901-eb6d2bcd4c7c)
![image](https://github.com/user-attachments/assets/cc56fc2c-884f-48b1-acf8-fdfd345b9156)

**Comment by chilingling** [comment, marked as answer]
左侧插件——资源管理。可以添加 npm 工具类或者 function 工具类，具体您可以看下官方文档。

![image](https://github.com/user-attachments/assets/6a96e81c-d7a1-4167-8996-be93cb1fb92b)
